edx/edx-enterprise
# ENT-399: Convert enterprise templates to django template

**Description:** This PR converts all mako templates in edx-enterprise to django templates

**JIRA:** [ENT-399](https://openedx.atlassian.net/browse/ENT-399)

**Dependencies:** https://github.com/edx/edx-platform/pull/15179

**Installation instructions:** Simply install edx-enterprise no other setup required.

**Testing instructions:**

1. Open data sharing consent page
2. Make sure it works as expected i.e. there are no errors
3.Open enterprise landing page
4.Make sure it works as expected i.e. there are no errors

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
